backend/data/models.py

```diff
@@ -69,76 +70,131 @@ class PersonName(Field, models.Model):
         to=Person,
         on_delete=models.CASCADE,
         null=False,
-        related_name='names',
+        related_name="names",
         unique=True,
     )
 
     def __str__(self):
         return self.value
 
 
+class CaseStudy(models.Model):
+    """
+    A case study is an overarching collection of epistolary events, bound together by a common theme, e.g. `The Saga of St. Boniface` or `The  Nun Rebellion of Poitiers`.
+    """
+
+    class Meta:
+        verbose_name = "case study"
+        verbose_name_plural = "case studies"
+
+    name = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+    )
+
+    def __str__(self):
+        return self.name
+
+
 class EpistolaryEvent(models.Model):
+    """
+    Epistolary events are groups of related letter actions that are connected in some way.
+
+    For instance, a political campaign (epistolary event) may consist of the writing, transporting and reading of individual letters (letter actions).
+    """
+
```

Comment:
I considered making EpistolaryEvents nestable, so one epistolary event (e.g. 'the Election of Pope Clement III') can have multiple Epistolary Events (e.g. 'Council of Troyes' and 'Murder of Baldwin of Nantes') etc., but we should probably ask the team first before we needlessly add more layers of complexity.

Reply:
I had the same idea. Indeed, let's do it like this for now, and check how the team wants to connect events. I can also imagine horizontal connections rather than hierarchical ones.